tests/unittest/_torch/multi_gpu/test_kv_pool_rebalance_tp.py

```
@@ -0,0 +1,330 @@
+)
+@pytest.mark.parametrize("world_size", [2, 4], ids=lambda x: f"cp:{x}")
+def test_cp_ranks_agree_on_rebalance_trigger(world_size, case):
+    """Pure CP (``tp_size == 1``) must agree just as TP does."""
+    _skip_if_not_enough_gpus(world_size)
+    flags = _flags_for(case, world_size)
+    expected = flags[0]
+
+    with MPIPoolExecutor(max_workers=world_size) as ex:
+        results = ex.map(
+            run_single_rank,
+            *zip(*[(world_size, run_cp_agreement, flags, expected)] * world_size),
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)